code/js/modules/Sitelist.js

```diff
@@ -154,6 +154,7 @@
       "spotify": { name: "Spotify Web Player", url: "http://www.spotify.com" },
       "spreaker": { name: "Spreaker", url: "http://www.spreaker.com" },
       "stitcher": { name: "Stitcher", url: "http://www.stitcher.com" },
+      "studiolink": { name: "StudioLink", url: "http://localhost:52992", alias: ["localhost"] },
```

Comment:
This is an interesting use case, I like it!

The URL and alias here need to change though. What this line will do now is inject the `studiolink` controller into every url matching `localhost` which we do not want to happen :)

Streamkeys has custom alias functionality on the extention's option page. So that is where this alias should be set (on a user basis). 

Change this to the general studiolink URL so it won't be injected into every user's localhost and remove the alias.